packages/salesforcedx-sobjects-faux-generator/src/generator/fauxClassGenerator.ts

```diff
@@ -4,10 +4,12 @@
 * Licensed under the BSD 3-Clause license.
 * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
 */
-import * as fs from 'fs';
-import * as path from 'path';
+import { LocalCommandExecution } from '@salesforce/salesforcedx-utils-vscode/out/src/cli';
 
```

Comment:
Do *not* have a new line here. All imports need to be back-to-back for the TSLint ordered-import rules to work (https://palantir.github.io/tslint/rules/ordered-imports/).

Reply:
will do - just slipped through